_layouts/partners/faq.html

```diff
@@ -0,0 +1,127 @@
+---
+layout: base
+---
+<header>
+  {% include skip_nav.html %}
+  {% include banner.html %}
+  {% include partners/header.html %}
+</header>
+<script>
+  function closeAllAccordions(){
+    // changes accordion button look through aria controls
+    let accordionButtons = document.getElementsByClassName('usa-accordion__button');
+    Array.from(accordionButtons).forEach(el => {
+      el.setAttribute("aria-expanded", "false");
+    });
+    // hides accordion content panels
+    let accordionContentPanels = document.getElementsByClassName('usa-accordion__content');
+    Array.from(accordionContentPanels).forEach(el => {
+      el.hidden = true;
+    })
+  };
+</script>
+<main id="main-content">
+    <section>
+        <div class="container partners-container">
+            <div class="grid-row">
+                <div class="tablet:grid-col-9 accordion-container">
+                  <div class="partners-body partners-title">{{ page.title | markdownify }}</div>
```

Comment:
If we're going with your suggestion to fix the tabs,  this will need to change to an h1 and get rid of the markdownify. # needs to be removed in the .md file as well.

Reply:
You can also get rid of the partners-body class